metron-interface/metron-alerts/src/app/model/alert.ts

```diff
@@ -0,0 +1,44 @@
+export class Alert {
```

Comment:
I haven't been a part of the review a whole lot, so forgive me if I'm asking a question that's already been answered.  This `Alert` class seems very Snort specific, but `is_alert` can potentially show up as a result of any of our input data.  Shouldn't this be relatively abstract and limited to the fields that are essential for the alert?